src/app/interceptors/error-handling.interceptor.spec.ts

```diff
@@ -254,4 +258,55 @@ describe('ErrorHandlingInterceptor', () => {
         expect(refreshSpy).toHaveBeenCalledTimes(1);
         expect(notificationService.apiError).toBeFalsy();
     });
+
+    it('should handle non-response Raiden API errors for path url requests', () => {
+        raidenConfig.api = '/api/v1';
+        // Unlike the HttpClientModule, the HttpClientTestingModule does not prepend the url
+        // property of a HttpResponse with the host if the request was made by a URL path.
+        const requestUrl = window.location.origin + raidenConfig.api;
+        const errorSpy = jasmine.createSpy();
+
+        service.makeRequest(requestUrl).subscribe(() => {
+            fail('On next should not be called');
+        }, errorSpy);
+
+        const request = httpMock.expectOne(requestUrl);
+        request.flush(
+            {},
+            {
+                status: 504,
+                statusText: '',
+            }
+        );
+
+        expect(notificationService.addErrorNotification).toHaveBeenCalledTimes(
```

Comment:
Does this only happen for Raiden API errors? Else I don't see how this test tests the new behavior. Maybe you should add what you do in line 307 with truthy here?

Reply:
Yes, only for Raiden API errors a notification should be shown. But adding `expect(notificationService.apiError).toBeTruthy();` definitely makes sense here. Thanks!